mm2src/mm2_main/src/lp_wallet.rs

```diff
@@ -413,40 +414,41 @@ pub struct GetMnemonicResponse {
     pub mnemonic: MnemonicForRpc,
 }
 
-#[derive(Debug, Display, Serialize, SerializeErrorType)]
+#[derive(Debug, Display, Serialize, SerializeErrorType, EnumFromStringify)]
 #[serde(tag = "error_type", content = "error_data")]
-pub enum GetMnemonicError {
+pub enum MnemonicsError {
     #[display(fmt = "Invalid request error: {}", _0)]
     InvalidRequest(String),
     #[display(fmt = "Wallets storage error: {}", _0)]
     WalletsStorageError(String),
     #[display(fmt = "Internal error: {}", _0)]
     Internal(String),
+    #[display(fmt = "Invalid password error: {}", _0)]
+    #[from_stringify("MnemonicError")]
```

Comment:
We have `MnemonicError` and now `MnemonicsError`, which are too similar.
Maybe `MnemonicsError` -> `ApiMnemonicError`?

Reply:
should be `MnemonicRpcError`... thanks